WalletWasabi.Fluent/ViewModels/CoinControl/SelectCoinsDialogViewModel.cs

```diff
@@ -26,10 +28,15 @@ namespace WalletWasabi.Fluent.ViewModels.CoinControl;
 	NavigationTarget = NavigationTarget.DialogScreen)]
 public partial class SelectCoinsDialogViewModel : DialogViewModelBase<IEnumerable<SmartCoin>>
 {
-	public SelectCoinsDialogViewModel(WalletViewModel walletViewModel)
+	private readonly CompositeDisposable _disposables = new ();
```

Comment:
I think we should move the creation of `Source` into `OnNavigatedTo()` so that the `CompositeDisposable` of it could be used.
Maybe it is better to merge this PR, then refactor it in a separate PR one.

Reply:
Maybe obsolete with https://github.com/zkSNACKs/WalletWasabi/pull/9774#discussion_r1047144168

Reply:
I think we don't need to do that. The ViewModel is recreated every time we lead the coin control view. Moving the initialization to `OnNavigatedTo()` isn't good for invariants.

Reply:
How about just simply call `Source.Dispose();` in the `OnNavigatedFrom()`? Having an individual `CompositeDisposable` just for this feels unnecessary.